.pipelines/mirror-aro-to-int.yml

```diff
@@ -37,8 +37,14 @@ jobs:
       ## set the variable
       echo "##vso[task.setvariable variable=TAG]${TAG}"
   - script: |
+      USERNAME=`echo "$(aro-v4-ci-pd-pull)" | base64 -d | cut -d':' -f1`
+      PASSWORD=`echo "$(aro-v4-ci-pd-pull)" | base64 -d | cut -d':' -f2-`
+
       az acr login --name "${{variables.rpIntImageAcr}}"
       az acr import \
+        --force \
```

Comment:
Why are you adding a force tag @bennerv ? Well great question.

Our INT e2e runs on master merge. What that does is build the image off of master and deploy to INT. If you tag the repo with a release tag v20220610 prior to it building the image, it will push a versioned image to INT instead of the short commit hash.

What that means is that the INT image may differ just slightly from the production image built during the release. So in order to ensure the images are the exact same, force push over the previous tag. This force push only happens in INT so it cannot affect existing images.